Formula/p/php.rb

```diff
@@ -304,11 +304,16 @@ def post_install
   end
 
   def caveats
```

Comment:
I'm not sure anyone will ever see this since CI is case insensitive and that will generate the caveats AFAIK.

Reply:
@tbussmann as @SMillerDev said: most of our users get their package definitions shipped by API, and that API is built through our CI: this means that they will never see the change in caveat (as the CI is case insensitive).

The only thing we could do is to remove the on_macos block and make the change unconditional. This also means Linux users will see. Linux is case-sensitive by default (but there are rare occasions where it is not). I guess adding it to Linux would not do any harm.